templates/gitlab.rb.j2

```diff
@@ -70,5 +70,17 @@ nginx['ssl_verify_client'] = "{{ gitlab_nginx_ssl_verify_client }}"
 nginx['ssl_client_certificate'] = "{{ gitlab_nginx_ssl_client_certificate }}"
 {% endif %}
 
+{% if gitlab_extra_settings is defined %}
+# Extra configuration
+{% for extra in gitlab_extra_settings %}
+{% for setting in extra %}
+{% for kv in extra[setting] %}
+{{ setting }}['{{ kv.key }}'] = '{{ kv.value }}'
+{% endfor %}
+{% endfor %}
+
```

Comment:
What do this empty line is used for ?

Reply:
This line is to create separated blocks of settings in the generated file.